gp-unique-id/gpuid-generate-post-workflow.php

```
@@ -39,6 +39,12 @@ public function init() {
 
 		add_filter( 'gpui_unique_id', array( $this, 'prevent_unique_id_generation' ), 10, 3 );
 		add_action( 'gravityflow_step_complete', array( $this, 'maybe_generate_unique_id' ), 10, 4 );
+		add_filter( 'gpui_unique_id', function( $unique, $form_id, $field_id ) {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$form_id'. (undefined)

(UnusedFormalParameter)